### ISSUE TITLE
Remove all vxTwitter mentions in README

### ISSUE THREAD

**Issue by MonfGeiger** [dylanpdx/BetterTwitFix#205]
I only say this because at the time of this post, i notice that all the urls for twitter.com i got bookmarked now show it as X(dot)com......and dear god i hate that its truly a permanent thing now.

I'm not saying to rename the entire project here, but since the [REDACTED] at [REDACTED] have chosen to push this as a permanent thing, vxTwitter now no longer works in discord for embed reasons, only fixvx works now. Also, slightly unrelated, but apparently the entire website refuses to work if i have enhanced tracking protection turned on for it in firefox, so YAY, another unwanted and stupid change they forced through :D (said no one ever)

**Comment by RAMADAN1900** [comment]
Big thx ❤️❤️❤️

**Comment by MonfGeiger** [comment]
yeah, i just tested it ouyt for myself and saw it still works, my bad

I was running off of an empty tank with a splitting headache plus a dog that kept playing the "go outside/want inside" game. ignore this thread as a result of my Nat 1 Perception check
